### ISSUE TITLE
Accessibility: Icon failing 4.1.2 Name, Role, Value

### ISSUE THREAD

**Issue by EthanMendel** [microsoft/vscode#300861]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version: 1.111.0
- OS Version: macOS 26.3.1 (25D2128)

Two accessibility issues were identified related to incorrect ARIA usage using toggler developer tool.
> [link for 4.1.2](https://www.ibm.com/able/requirements/requirements/?version=v7_3#4_1_2)

Issue 1: A `<div>` element with an implicit generic role uses aria-label, which is invalid and ignored by assistive technologies.
> The ARIA attributes "aria-label" are not valid for the element `<a>` with implicit ARIA role "generic"

Issue 2: An interactive element with role="tab" does not have a programmatically associated accessible name (empty aria-label), causing screen readers to announce the tab without a label.
> Interactive component with ARIA role 'tab' does not have a programmatically associated name

<img width="3436" height="2204" alt="Image" src="https://github.com/user-attachments/assets/a4e9b9b5-019a-4fba-81b4-e510a8bef3d1" />
